<launch>
  <node name="graph_optimisation" pkg="crowdbot_active_slam"
    type="graph_optimisation" output="screen">
    <param name="node_dist_linear" value="0.1"/>
    <param name="node_dist_angular" value="0.175"/>
  </node>
</launch>
